src/solvers/flattening/boolbv_width.cpp

```diff
@@ -206,6 +206,10 @@ const boolbv_widtht::entryt &boolbv_widtht::get_entry(const typet &type) const
   {
     cache_entry = defined_entryt{0};
   }
+  else if(type_id == ID_rational)
```

Comment:
```suggestion
  else if(type_id == ID_rational || type_id == ID_real || type_id == ID_integer)
```
I added this line because we needed arrays of rationals, but should probably also support arrays of reals and integers.